lib/locations/locations.go

```diff
@@ -49,7 +49,6 @@ func init() {
 		panic(err)
 	}
 }
-
```

Comment:
why?

Reply:
Don't know, not to check your (quite impressive) attentiveness.